docs/README-AWS.md

```diff
@@ -398,70 +479,99 @@ inputs:
 
-## Routing support for AWS Services Logs
-When using Elastic integrations, as a first step users should install appropriate [integration](https://docs.elastic.co/en/integrations) assets using the Kibana UI. This sets up appropriate pre-built dashboards, ingest node configurations, and other assets that help you get the most out of the data you ingest. The integrations use [data streams](https://www.elastic.co/guide/en/elasticsearch/reference/current/data-streams.html) with specific [naming conventions](https://www.elastic.co/blog/an-introduction-to-the-elastic-data-stream-naming-scheme) providing users with more granular controls and flexibility on managing the ingested data.
+* AWS CloudTrail (`aws.cloudtrail`)
+* Amazon CloudWatch (`aws.cloudwatch_logs`)
+* Elastic Load Balancing (`aws.elb_logs`)
+* AWS Network Firewall (`aws.firewall_logs`)
+* Amazon VPC Flow (`aws.vpcflow`)
+* AWS Web Application Firewall (`aws.waf`)
 
-For `S3 SQS Event Notifications input` the Lambda function supports automatic routing of several AWS service logs to the corresponding [integration](https://docs.elastic.co/en/integrations) [data streams](https://docs.elastic.co/en/integrations) for further processing and storage in the Elasticsearch cluster. It supports automatic routing of AWS CloudTrail (`aws.cloudtrail`), Amazon CloudWatch Logs (`aws.cloudwatch_logs`), Elastic Load Balancing(`aws.elb_logs`), AWS Network Firewall (`aws.firewall_logs`), Amazon VPC Flow (`aws.vpcflow`) & AWS Web Application Firewall (`aws.waf`) logs to corresponding default integrations data streams. Setting the `es_datastream_name` field in the configuration file is optional for this use case.
+**Notes:**
 
-For most of the other use cases, the user will need to set the `es_datastream_name` field in the configuration file to route the data to a specific data stream or an index. This value should be set in the following use cases:
-- Users want to write the data to a specific index, alias or a custom data stream and not to the default integration data streams. This can help some users to use the existing Elasticsearch setup like index templates, ingest pipelines or dashboards that you may have already set up and may have developed a business process around it and don’t want to change it.
-- When using `Kinesis Data Stream`, `CloudWatch Logs subscription filter` or `Direct SQS message payload` input. Only `S3 SQS Event Notifications input` method supports automatic routing to default integrations data streams for several AWS services logs.
-- When using `S3 SQS Event Notifications input` but the log types is something other than AWS CloudTrail (`aws.cloudtrail`), Amazon CloudWatch Logs (`aws.cloudwatch_logs`), Elastic Load Balancing (`aws.elb_logs`), AWS Network Firewall (`aws.firewall_logs`), Amazon VPC Flow (`aws.vpcflow`) & AWS Web Application Firewall (`aws.waf`).
+For these use cases, setting the `es_datastream_name` field in the configuration file is optional.
```

Comment:
uhm... I'm not sure saying that the `es_datastream_name` field is optional is the proper definition.
If we don't set it, and the input is an `S3 SQS Event Notifications`, the forwarder will try to route the content to the corresponding service logs.
but if it is set, no automatic routing will be done.
they are more two mutually exclusive options: either setting the `es_datastream_name` field or relying on the automatic routing